common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -1189,13 +1198,14 @@ def test_get_items(self):
         locator = CourseLocator(org='testx', course='GreekHero', run="run", branch=BRANCH_NAME_DRAFT)
         # get all modules
         matches = modulestore().get_items(locator)
-        self.assertEqual(len(matches), 7)
+        self.assertEqual(len(matches), 8)
         matches = modulestore().get_items(locator)
-        self.assertEqual(len(matches), 7)
+        self.assertEqual(len(matches), 8)
         matches = modulestore().get_items(locator, qualifiers={'category': 'chapter'})
-        self.assertEqual(len(matches), 3)
+        self.assertEqual(len(matches), 4)
         matches = modulestore().get_items(locator, qualifiers={'category': 'garbage'})
         self.assertEqual(len(matches), 0)
+        # Test that we don't accidentally get an item with a similar name.
         matches = modulestore().get_items(locator, qualifiers={'name': 'chapter1'})
         self.assertEqual(len(matches), 1)
```

Comment:
@haikuginger Isn't a test like below necessary to ensure a substring match is no longer performed?:
```
matches = modulestore().get_items(locator, qualifiers={'name': 'chap})
self.assertEqual(len(matches), 1)
```